src/histolab/tiler.py

```diff
@@ -410,7 +413,9 @@ def extract(self, slide: Slide):
         tiles_counter = 0
         for tiles_counter, (tile, tile_wsi_coords) in enumerate(random_tiles):
             tile_filename = self._tile_filename(tile_wsi_coords, tiles_counter)
-            tile.save(tile_filename)
+            root_folder = slide.processed_path
```

Comment:
maybe this assignment is not mandatory
`os.path.join(self.processed_path, tile_filename)` is enough i believe